js/redux/modules/teams.js

```diff
@@ -178,13 +179,20 @@ export const createTeamAndAddUser = (
 ) => async dispatch => {
   try {
     const uid = await AsyncStorage.getItem("user");
+<<<<<<< HEAD
     firebaseDB
+=======
+    const name = teamName(type);
+    let teamId = "";
+    await firebaseDB
+>>>>>>> Add matches screen; add random team name when create new team.
```

Comment:
I guess you tried to remove some conflicts. Just make sure this is the code you want and remove the comments.

Reply:
Changes done! Please review :)